cmd/keymasterd/main.go

```diff
@@ -311,6 +362,13 @@ func (state *RuntimeState) checkAuth(w http.ResponseWriter, r *http.Request) (st
 		authCookie = cookie
 	}
 	if authCookie == nil {
+
+		if (AuthTypePassword & requiredAuthType) == 0 {
+			state.writeFailureResponse(w, r, http.StatusUnauthorized, "")
+			err := errors.New("Insufficeint Auth Level passwd")
```

Comment:
This is opaque to the user. What does this mean?

Reply:
To the use they get a 401 error page. For the logs it tell us that there was an a log-in attempt, but the url requested more than a password for login (and there were no cookies in the call).